bin/ansible-galaxy

```diff
@@ -703,7 +703,8 @@ def execute_info(args, options, parser):
                 del install_info['version']
             role_info.update(install_info)
 
-        remote_data = api_lookup_role_by_name(api_server, role, False)
+        remote_data = api_lookup_role_by_name(api_server=api_server, role_name=role,
```

Comment:
Explicit is better than implicit. Using this syntax avoid an error if someone add or remove an argument. (e.g. this PR with the parser argument)